### PR TITLE
Added test jan 25

### PR DESCRIPTION
**Pull request by manjunath-grl**
Added automated test:
TC-ACL-2.10
Simulated test:
TC-LTIME-1.1_Simulated
Added auto generated files

Tested

Execution logs:
[exe_log.txt](https://github.com/project-chip/connectedhomeip/files/10499511/exe_log.txt)
[LTIME 1.1.zip](https://github.com/project-chip/connectedhomeip/files/10499514/LTIME.1.1.zip)

### CODE REVIEW

**Review by bzbarsky-apple — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2021 Project CHIP Authors
```
Comment (yufengwangca) [inline src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:1]: 2023

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+config:
+    nodeId: 0x12344321
+    cluster: "Time Format Localization"
+    endpoint: 0
+
+tests:
+    - label: "Wait for the device to be commissioned"
+      cluster: "DelayCommands"
+      command: "WaitForCommissioning"
```
Comment (yufengwangca) [inline src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:29]: why no "nodeId" argument? which device to be commissioned?

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    - label: "DUT read (0xFFFD) ClusterRevision attribute"
+      PICS: LTIME.C.Afffd
+      wait: "readAttribute"
+      attribute: "ClusterRevision"
```
Comment (yufengwangca) [inline src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:34]: Add 'response' tag to verify the result

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+    - label: "DUT read (0xFFFC) FeatureMap attribute"
+      PICS: LTIME.C.Afffc
+      wait: "readAttribute"
+      attribute: "FeatureMap"
```
Comment (yufengwangca) [inline src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:39]: Add 'response' tag to verify the result

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    #Commenting out the step EventList attribute which is out of scope for matter V1.0
+    #- label: "DUT read (0xFFFA) EventList attribute"
+    #  PICS: LTIME.C.Afffa
+    #  wait: "readAttribute"
+    #  attribute: "EventList"
+
+    - label: "DUT read (0xFFF9) AcceptedCommandList attribute"
+      PICS: LTIME.C.Afff9
+      wait: "readAttribute"
+      attribute: "AcceptedCommandList"
```
Comment (yufengwangca) [inline src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:55]: Add 'response' tag to verify the result

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+    - label: "DUT read (0xFFF8) GeneratedCommandList attribute"
+      PICS: LTIME.C.Afff8
+      wait: "readAttribute"
+      attribute: "GeneratedCommandList"
```
Comment (yufengwangca) [inline src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:60]: Add 'response' tag to verify the result

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml (hunk 1 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:1]: i will update it.

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml (hunk 2 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:29]: @yufengwangca LTIME-1.1 is simulated test not requires nodeId arguments in YAML script.
Please refer doc: https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/simulated_device_linux.md

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml (hunk 3 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:34]: @yufengwangca In simulated tests response check for attribute read is not implemented in YAML.

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml (hunk 4 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:39]: @yufengwangca In simulated tests response check for attribute read is not implemented in YAML.

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+    - label: "DUT read (0xFFFC) FeatureMap attribute"
+      PICS: LTIME.C.Afffc
+      wait: "readAttribute"
+      attribute: "FeatureMap"
+
+    - label: "DUT read (0xFFFB) AttributeList attribute"
+      PICS: LTIME.C.Afffb
+      wait: "readAttribute"
+      attribute: "AttributeList"
```
[the remark this answers is not in the document]
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:44]: @yufengwangca In simulated tests response check for attribute read is not implemented in YAML.

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml (hunk 5 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:55]: @yufengwangca In simulated tests response check for attribute read is not implemented in YAML.

File: src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml (hunk 6 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml:60]: @yufengwangca In simulated tests response check for attribute read is not implemented in YAML.

**Review by raju-apple — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**